src/js/events.js

```diff
@@ -487,7 +487,7 @@ var Events;
         handleKeydown: function (event) {
             this.triggerCustomEvent('editableKeydown', event, event.currentTarget);
 
-            if (Util.isKey(event, Util.keyCode.ENTER)) {
+            if (Util.isKey(event, Util.keyCode.ENTER) || Util.isKey(event, Util.keyCode.ENTER_OF_EMACS)) {
```

Comment:
Currently, this would happen anytime you hit <kbd>M</kbd> on the keyboard.  As per the description in your pull request, you were looking to target <kbd>CTRL</kbd> + <kbd>M</kbd> right?

If so, you'll want to add the check for <kdb>CTRL</kbd>.  There's a `Util.isMetaCtrlKey()` helper function you can use here, so you can end up with something like:

```js
if (Util.isKey(event, Util.keyCode.ENETER) || 
        (Util.isMetaCtrlKey(event) && Util.isKey(event, Util.keyCode.M))) {
    // ...
}
```